patch.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/app/i18n/pt/strings.xml b/app/i18n/pt/strings.xml
+index 7dafbd5..6207fa7 100644
+--- a/app/i18n/pt/strings.xml
++++ b/app/i18n/pt/strings.xml
```

Comment:
Not sure what this is, but looks like something with your git client went really wrong. The relevant files have not been changed, but there is a `patch.diff` here. The other changes seem to come from an outdated master-fork. Can I commit them manually or do you want to do a new PR?